### DISCUSSION TITLE
error "The maximum number of workspaces has been reached"

### DISCUSSION THREAD

**Discussion by phankhanhpvk** [outline/outline#5799] [Self Hosting]
I setup google and slack authentication method, when login with slack i get error `The maximum number of workspaces has been reached`
`{"error":"The maximum number of workspaces has been reached","level":"error","message":"Error during authentication","stack":"BadRequestError: The maximum number of workspaces has been reached\n    at MaximumTeamsError (/mnt/outline/build/server/errors.js:126:34)\n    at teamProvisioner (/mnt/outline/build/server/commands/teamProvisioner.js:74:43)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}`

**Comment by tommoor** [comment]
Did you set allowed domains rule that would prevent the user from joining the existing team?

**Reply by phankhanhpvk** [reply]
Yes, i configed

**Reply by phankhanhpvk** [reply]
please let me know how i can fix this

**Reply by tommoor** [reply]
The error is because the user cannot join the existing workspace due to rules you've set and only one workspace can exist in community edition. The answer is probably to invite the user first or change the allowed domains rules

**Reply by DalenW** [reply]
@tommoor I have the same issue. Setup just google authentication and enabled email authentication, now users can only sign in with email. Attempting to sign in with Google throws this error, on new and current users. The domain is definitely allowed. Best I can tell is that it just can't find a team for the user.

**Reply by gakest2000** [reply]
Hello @tommoor I have exactly the same issue. Did you mean that Google  and slack authentication cannot co-exist in the community edition?
And please how to check the allowed domain settings.
Thanks

**Reply by tommoor** [reply]
Please see the docs here for details on allowed domains:
https://docs.getoutline.com/s/guide/doc/authentication-Wr4sfjvmL1#h-allowed-domains

Try inviting the user specifically and see if that allows them to sign-in
